core/lexicon/en/file.inc.php

```diff
@@ -73,6 +73,7 @@
 $_lang['file_folder_remove'] = 'Delete Directory';
 $_lang['file_folder_remove_confirm'] = 'Are you sure you want to delete the directory: "[[+directory]]"?<br />This could potentially break your website.';
 $_lang['file_folder_rename'] = 'Rename Directory';
+$_lang['file_info_metadata_new'] = 'Unable to access metaData for the file “[[+file]]” because it is for a new element and has not been created yet. An attempt will be made to create the file automatically.';
```

Comment:
This isn't still necessary since the isNew param was removed, correct?

Reply:
Yes, forgot that! Just removed it in new commit.